frontend/src/bundles/resume/components/resume-editor/resume-editor-container/resume-editor-container.tsx

```diff
@@ -0,0 +1,16 @@
+import { type LayoutContainer } from '~/bundles/resume/types/types';
+
+import { ResumeEditorBlock } from '../resume-editor-block/resume-editor.block';
+
+const ResumeEditorContainer: React.FC<LayoutContainer> = ({
```

Comment:
I think I made these components already) check RegularItem/Block/Container

Reply:
y, regular block could be reused)